proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -317,7 +321,7 @@ public void operationComplete(ChannelFuture future) throws Exception
                         }
                     }
                 };
-                new RemoteQuery( this, info ).start( PipelineUtils.getDatagramChannel(), new InetSocketAddress( info.getHost().getAddress(), info.getQueryPort() ), eventLoops, bindListener );
+                new RemoteQuery( this, info ).start( PipelineUtils.getDatagramChannel(), new InetSocketAddress( info.getHost().getAddress(), info.getQueryPort() ), workerEventLoop, bindListener );
```

Comment:
We should probably have `RemoteQuery` accept another `EventLoopGroup` parameter here for the boss group.